packages/components/psammead-media-player/src/Amp/index.jsx

```diff
@@ -24,7 +24,12 @@ const AmpMediaPlayer = ({ src, placeholderSrc, title }) => {
         title={title}
         allowfullscreen="allowfullscreen"
       >
-        <amp-img layout="fill" src={placeholderSrc} placeholder />
+        <amp-img
+          layout="fill"
+          src={placeholderSrc}
+          srcset={placeholderSrcset}
+          placeholder
+        />
```

Comment:
Just noticed we have `psammead-image/AmpImg`, might it be better to use it here isntead of `amp-img` directly for consistency? https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image/README.md#ampimg-1